nixos/modules/services/networking/dnschain.nix

```diff
@@ -32,77 +40,139 @@ in
 
     services.dnschain = {
 
-      enable = mkOption {
-        type = types.bool;
-        default = false;
+      enable = mkEnableOption ''
+        DNSChain, a blockchain based DNS + HTTP server.
+        To resolve .bit domains set <literal>services.namecoind.enable = true;</literal>
+        and an RPC username/password.
+      '';
+
+      dns.address = mkOption {
+        type = types.string;
```

Comment:
Probably should be `types.str` unless you want the peculiar merge semantics of `string`